doc/release/upcoming_changes/15355.c_api.rst

```diff
@@ -0,0 +1,8 @@
+Const qualify UFunc inner loops
+-------------------------------
+`UFuncGenericFunction` now expects pointers to const ``dimension`` and

```

Comment:
```suggestion
``UFuncGenericFunction`` now expects pointers to const ``dimension`` and
```
probably breaks release notes build otherwise. But its currently failing anyway, so cannot see (have to mop that up). In any case, looks good to me. Should merge this soon.